src/components/dataset/overview/new-property.vue

```diff
@@ -91,7 +100,10 @@ const submit = () => {
       "To add an Entity property, choose a unique property name below.",
       "You can also add new properties by uploading a Form that references them, in which case the properties are created when the Form is published."
     ],
-    "newPropertyName": "New property name"
+    "newPropertyName": "New property name",
+    "problem": {
+      "409_3": "A property already exists in this Entity List with the name of “{propertyName}”."
```

Comment:
I usually just use quotes for strings that might contain a space. Since `propertyName` will be a single word, what do you think about removing the quotes here? No strong feelings!

Reply:
I think the quotes isolate the name better than without them, since there is no other formatting.